app/scripts/filters/resources.js

```diff
@@ -1355,4 +1355,9 @@ angular.module('openshiftConsole')
   })
   .filter('humanizePodStatus', function(humanizeReasonFilter) {
     return humanizeReasonFilter;
+  })
+  .filter('hpaWarningsForResource', function() {
```

Comment:
This is a very specific filter. I'd give it a name that makes it clear what it's really doing like `rejectV2HPAWarnings`. But we might not need it. (See my comments below.)

Reply:
Ok I can remove.